### PR TITLE
Display reviewers on description page

### PR DESCRIPTION
**Pull request by RMacfarlane**
reflow-sidebar

Displays reviewers and labels in a sidebar. Will tackle adding new reviewers and labels in a separate PR

### CODE REVIEW

File: src/github/pullRequestOverview.ts [hunk 1]
```diff
@@ -133,6 +134,31 @@ export class PullRequestOverviewPanel {
 		}
 	}
 
+	private parseReviews(pullRequestModel: PullRequestModel, timelineEvents: TimelineEvent[]): ReviewState[] {
+		const reviewEvents = timelineEvents.filter(isReviewEvent);
+		const reviews: ReviewState[] = [];
+		const seen = new Map<string, boolean>();
+		for (let i = reviewEvents.length -1; i >= 0; i--) {
+			const reviewer = reviewEvents[i].user;
+			if (!seen.get(reviewer.login)) {
```
Comment (rebornix) [inline src/github/pullRequestOverview.ts]: We need to filter out pull request creator as their comments are being labeled as reviews as well.

File: src/github/pullRequestOverview.ts [hunk 2]
```diff
@@ -133,6 +134,31 @@ export class PullRequestOverviewPanel {
...
+				seen.set(reviewer.login, true);
+				reviews.push({
+					reviewer: reviewer,
+					state: reviewEvents[i].state as any
+				});
+			}
+		}
+
+		pullRequestModel.prItem.reviewRequests.forEach(request => {
```
Comment (rebornix) [inline src/github/pullRequestOverview.ts]: GitHub.com sorts the results by alphabetical order.

File: preview-src/index.ts [hunk 3]
```diff
@@ -145,57 +194,28 @@ function renderTitle(pr: PullRequest): HTMLElement {
...
-
-	commentContainer.appendChild(commentHeader);
-	commentContainer.appendChild(commentBody);
-
-	if (pr.canEdit) {
-		function updateDescription(text: string) {
+function renderDescription(pr: PullRequest): void {
+	const descriptionNode = document.getElementById('description');
+	descriptionNode.innerHTML = '';
+	const bodyHTML = !pr.body ? '<i>No description provided</i>' : pr.bodyHTML;
```
Comment (rebornix) [inline preview-src/index.ts]: REST API doesn't have bodyHTML, right?

File: preview-src/index.ts [hunk 4]
```diff
@@ -145,57 +194,28 @@ function renderTitle(pr: PullRequest): HTMLElement {
...
+	const descriptionElement = renderComment({
```
Comment (rebornix) [inline preview-src/index.ts]: is this intentional that now we render the description as a comment? I kind of like the ui before.

Comment (RMacfarlane) [reply preview-src/index.ts]: Do you mean having the information about the merge as the title of the description?
screen shot 2019-02-14 at 2 53 05 pm
screen shot 2019-02-14 at 2 54 11 pm

**Pull request merged**